src/js/components/modals/AppModalComponent.jsx

```diff
@@ -142,6 +142,12 @@ var AppModalComponent = React.createClass({
 
     var modelAttrs = Util.serializedArrayToDictionary(attrArray);
 
+    if (modelAttrs.cmd === "") {
+      // An outstanding bug in Marathon #2147 means that a cmd field cannot be
+      // overridden with a blank string without failing validation.
+      modelAttrs.cmd = " ";
```

Comment:
This allows me to create an application with neither a docker image specified, nor a command. To reproduce:

- new app
- enter id
- submit

the app gets created.